modules/odo-getting-started-delete.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-nodejs.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-dot-net.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-go.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-java.adoc
+
+:_content-type: Procedure
+[id="odo-getting-started-delete_{context}"]
+
+= Deleting the resources (optional)
```

Comment:
```suggestion
= Optional: Deleting the resources
```
OR
```suggestion
= Deleting the resources
```

[optional-in-headings.txt](https://github.com/openshift/openshift-docs/files/9471989/optional-in-headings.txt)

Do we really need to say "optional" in the title? 
Most procedures are _conditional_ and _optional_. That is, the abstract says tells the user that, to achieve result x, do y.

Reply:
The Getting Started guide is essentially a single procedure with multiple sub-procedures, some of which are required and some optional. Unfortunately, our "Modular Docs" standard doesn't allow nested modules :-(.  In the overall flow of the guide, it makes sense to identify which sub-procedures are optional, so going with the Optional: prefix here